src/transformers/trainer.py

```diff
@@ -1007,11 +1007,15 @@ def _wrap_model(self, model, training=True):
                 find_unused_parameters = not model.is_gradient_checkpointing
             else:
                 find_unused_parameters = True
+            kwargs = {}
```

Comment:
Maybe put the `find_unused_parameters` here and remove it from the wrapper below?

Reply:
Sure, done!